index.html

```diff
@@ -11,6 +11,9 @@
     <h1>Hello LIMA, how are you?</h1>
     <p>We are coding......happy coding</p>
     <h2>Hello world, We are software developers</h2>
-    
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore nam amet esse culpa corporis accusamus voluptatem officia fugit ut quas neque cum iure quasi, praesentium possimus saepe ab error. Maiores.</p>
```

Comment:
- [ ] I think It would be better if you redused the spacing /indentation on this line, because it  makes the code scattered and less professional